web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -1005,6 +1033,188 @@ function AdminRule({
   );
 }
 
+function Options({
+  value,
+  isProcessing,
+  onChange,
+}: SectionProps<OptionsModel, never>) {
+  const theme = useTheme();
+  const id = useId();
+  const maxSessionTTLId = `${id}-max-session-ttl`;
+  const clientIdleTimeoutId = `${id}-client-idle-timeout`;
+  const requireMFATypeId = `${id}-require-mfa-type`;
+  const createHostUserModeId = `${id}-create-host-user-mode`;
+  return (
+    <Box
+      border={1}
+      borderColor={theme.colors.interactive.tonal.neutral[0]}
+      borderRadius={3}
+      p={3}
+      css={`
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+        align-items: baseline;
+        row-gap: ${theme.space[3]}px;
+      `}
+    >
+      <H4
+        css={`
+          grid-column: 1/3;
+        `}
+      >
+        Global Settings
+      </H4>
+
+      <OptionLabel htmlFor={maxSessionTTLId}>Max Session TTL</OptionLabel>
+      <Input
+        id={maxSessionTTLId}
+        value={value.maxSessionTTL}
+        disabled={isProcessing}
+        onChange={e => onChange({ ...value, maxSessionTTL: e.target.value })}
+      />
+
+      <OptionLabel htmlFor={clientIdleTimeoutId}>
+        Client Idle Timeout
+      </OptionLabel>
+      <Input
+        id={clientIdleTimeoutId}
+        value={value.clientIdleTimeout}
+        disabled={isProcessing}
+        onChange={e =>
+          onChange({ ...value, clientIdleTimeout: e.target.value })
+        }
+      />
+
+      <div>Disconnect When Certificate Expires</div>
+      <BoolRadioGroup
+        name="disconnect-expired-cert"
+        value={value.disconnectExpiredCert}
+        onChange={d => onChange({ ...value, disconnectExpiredCert: d })}
+      />
+
+      <OptionLabel htmlFor={requireMFATypeId}>Require Session MFA</OptionLabel>
+      <Select
+        inputId={requireMFATypeId}
+        isDisabled={isProcessing}
+        options={requireMFATypeOptions}
+        value={value.requireMFAType}
+        onChange={t => onChange?.({ ...value, requireMFAType: t })}
+      />
+
+      <H4
+        css={`
+          grid-column: 1/3;
+          border-top: ${theme.borders[1]}
+            ${theme.colors.interactive.tonal.neutral[0]};
+          padding-top: ${theme.space[3]}px;
+        `}
+      >
+        SSH
+      </H4>
```

Comment:
Seeing this H4 grid repeated with only the text values (SSH,Database, Desktop etc)  being unique, I think you will gain by programmatically creating each row instead of with hand. Given the scope of our role option, this will likely grow a larger list  and I think will be cleaner that way.